apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -72,7 +72,13 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
         state?.config?.ui?.isPasswordInputValidationEnabled);
 
     const predefinedCategories: any = useMemo(() => {
-        const originalConnectors: any = GovernanceConnectorUtils.getPredefinedConnectorCategories();
+        let originalConnectors: Array<any> = GovernanceConnectorUtils.getPredefinedConnectorCategories();
+
+        if (!featureConfig?.organizationDiscovery?.enabled) {
+            originalConnectors = originalConnectors.filter(
+                (category: any) => category.id !== ServerConfigurationsConstants.ORGANIZATION_SETTINGS_CATEGORY_ID
```

Comment:
refrain from using `any`

Reply:
In this instance, we have to use `any` since `originalConnectors` array is of type `Array<any>`. This is because the value returned by `GovernanceConnectorUtils.getPredefinedConnectorCategories()` method is of type `Array<any>` and is assigned to variable `originalConnectors`.